src/components/Reports/ReportsDailyVisits/ReportsDailyVisits.js

```diff
@@ -18,23 +18,71 @@ const ReportsDailyVisits = ({ domainName, dateFrom, dependencies: { datahubClien
       show: false,
     },
     tooltip: {
-      format: {
-        title: (date) => {
-          return intl.formatDate(date, {
+      contents: function(data) {
+        if (data.length) {
+          const date = intl.formatDate(data[0].x, {
             timeZone: 'UTC',
             day: 'numeric',
             month: 'long',
             year: 'numeric',
             weekday: 'long',
           });
-        },
-        value: (value) => {
-          return (
-            intl.formatMessage({
-              id: 'reports_daily_visits.tooltip_page_views',
-            }) + value
-          );
-        },
+          let quantity = '',
+            withEmail = '',
+            withoutEmail = '';
+          for (const item of data) {
+            switch (item.id) {
+              case 'quantity':
+                quantity =
+                  intl.formatMessage({
+                    id: 'reports_daily_visits.tooltip_page_views',
+                  }) + item.value;
+                break;
+              case 'withEmail':
+                withEmail =
+                  item.value !== null
+                    ? intl.formatMessage({
+                        id: 'reports_daily_visits.tooltip_with_email',
+                      }) + item.value
+                    : '';
+                break;
+              case 'withoutEmail':
+                withoutEmail =
+                  item.value !== null
+                    ? intl.formatMessage({
+                        id: 'reports_daily_visits.tooltip_without_email',
+                      }) + item.value
+                    : '';
+                break;
+              default:
+                return '';
+            }
+          }
+          return `
+          <table class="c3-tooltip">
```

Comment:
Why use table?

Reply:
Because this is the structure already used by the library c3, some styles are tied to this. I could change them for this case, making a simple structure.